recipes/eccodes-python/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "eccodes-python" %}
+{% set version = "0.9.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 587dd828ee5e13b6a7d35a362d6fd903d914dd0422f4010af95e8f45738e9143
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - numpy
+    - python
+    - cffi
+    - attrs
+    - eccodes
+
+test:
+  imports:
+    - gribapi
+    - eccodes
+
+about:
+  home: https://github.com/ecmwf/eccodes-python
+  license: Apache-2.0
+  license_family: Apache
+  license_file: LICENSE.txt
```

Comment:
Note that this file name is relative to the package root, not relative to the recipe. It is therefore not necessary to include the license with the recipe. Therefore this should be `LICENSE` to match the file name in the source dist.